static_src/components/navbar.jsx

```diff
@@ -23,23 +27,33 @@ export class Nav extends React.Component {
     this.props = props;
     this.state = {
       currentOrg: OrgStore.get(this.props.initialCurrentOrgGuid),
-      orgs: []
+      currentSpace: SpaceStore.get(this.props.initialSpaceGuid),
+      orgs: OrgStore.getAll() || []
     };
     this._onChange = this._onChange.bind(this);
+    this._handleOverviewClick = this._handleOverviewClick.bind(this);
   }
 
   componentDidMount() {
     OrgStore.addChangeListener(this._onChange);
+    SpaceStore.addChangeListener(this._onChange);
   }
 
   componentWillUnmount() {
     OrgStore.removeChangeListener(this._onChange);
+    SpaceStore.removeChangeListener(this._onChange);
   }
 
   _onChange() {
     this.setState(stateSetter());
   }
 
+  _handleOverviewClick() {
+    // orgActions.changeCurrentOrg('0');
+    orgActions.toggleSpaceMenu('0');
```

Comment:
could this be `toggleSpaceMenu` or change the following line to `toggleCurrentSpace`